tests/10-issue.js

```diff
@@ -26,95 +26,284 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
-          const issuer = issuers.find(issuer => issuer.tags.has('Revocation'));
+          const issuer = issuers.find(
+            issuer => issuer.tags.has('BitstringStatusList'));
           const {result, error, data} = await issueVc({issuer});
           err = error;
           issuerResponse = result;
           issuedVc = data;
+          const credentialStatusType = typeof (issuedVc.credentialStatus);
+          if(credentialStatusType === 'object') {
+            // TODO make everything an array and test each status entry
+            // issuedVc.credentialStatus = [issuedVc.credentialStatus]
+          }
         });
-        it('MUST issue a VC with BitstringStatusList "credentialStatus" and ' +
-          '"revocation" status purpose', async function() {
-          this.test.cell = {columnId: issuerName, rowId: this.test.title};
-          should.exist(issuerResponse);
-          should.not.exist(err);
-          issuerResponse.status.should.equal(201);
-          should.exist(issuedVc, `Expected VC from ${issuerName} to exist.`);
-          testCredential({credential: issuedVc});
-        });
-        // ensure that issued StatusList Credential contain correct properties
-        it('MUST have correct properties when dereferencing ' +
-          '"credentialStatus.statusListCredential"', async function() {
-          this.test.cell = {columnId: issuerName, rowId: this.test.title};
-          const {slc} = await getSlc({issuedVc});
-          testSlCredential({slCredential: slc});
+        describe('BitstringStatusList Entry', function() {
+          it('Any expression of the data model in this section ' +
+            'MUST be expressed in a conforming verifiable credential ' +
+            'as defined in [VC-DATA-MODEL-2.0].',
+          async function() {
+            should.exist(issuerResponse,
+              'Expected an issuer response.');
+            should.not.exist(err,
+              'Expected no errors in the issuer response.');
+            issuerResponse.status.should.equal(201,
+              'Expected response http code 201.');
+            should.exist(issuedVc, `Expected VC from ${issuerName} to exist.`);
+            testCredential({credential: issuedVc});
+          });
+          it('If present, the id value is expected to be a URL ' +
+            'that identifies the status information associated ' +
+            'with the verifiable credential. ' +
+            'It MUST NOT be the URL for the status list.',
+          async function() {
+            if(!!issuedVc.credentialStatus.id) {
+              // TODO test for URI
+              issuedVc.credentialStatus.id.should.be.a('string',
+                'Expected credentialStatus.id to be a string.'
+              );
+            } else {
+              this.skip();
+            }
+          });
+          it('The type property MUST be BitstringStatusListEntry. ',
+            async function() {
+              issuedVc.credentialStatus.hasOwnProperty('type',
```

Comment:
is this a valid chai assertion? it looks like you're using a basic javascript api: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty

maybe you should assert on the return of hasOwnProperty?

Reply:
removed all occurrences of `hasOwnProperty` in favor of `should.have.own.property`